### ISSUE TITLE
`ModelWrap`'s `normalize_model` changes the the model structure

### ISSUE THREAD

**Issue by lsrosa** [SSIGPRO/peepholelib#152]
Adding the normalization layer changes the model's submodules architecture, which becomes {'normalized', 'model'}.
This makes the output layer to become `'model'.<output_layer>`.

While this is not a problem, it breaks if we call `normalize_model()` before `update_output()`, which should not be the case since these are independent.

I would suggest keeping the normalization layer inside `ModelWrap`, with a default to `lambda x: x` and add the normalization computation to `ModelWraps.__forward__()`, so the model's `state_dict` remains unchanged.

**Comment by lsrosa** [comment]
This is also creating inconsistencies with loading the model's parameters if the model is saved with or without the normalization. We need to change this and retrain the models trained with the normalization.

**Comment by lsrosa** [comment]
This one is solved already.
